src/components/engagement/handlers/apply-finalized-changeset-to-engagement.handler.ts

```diff
@@ -61,21 +62,28 @@ export class ApplyApprovedChangesetToEngagement
               relation('in', 'changesetRel', 'changeset', ACTIVE),
               node('changeset'),
             ])
-            .setValues({
-              'engagementRel.active': true,
-            })
+            .apply((q) =>
+              status === ProjectChangeRequestStatus.Approved
+                ? q.setValues({
+                    'engagementRel.active': true,
+                  })
+                : q.apply(rejectChangesetProps())
```

Comment:
Is it weird that we _reject_ here but _commit_ below? it feels like there should be some symmetry with this logic.

Reply:
I think it will be more clear to reject [here](https://github.com/SeedCompany/cord-api-v3/blob/e6113e787bb30618ab4cc5ce30595db9f9cb045c/src/components/engagement/handlers/apply-finalized-changeset-to-engagement.handler.ts#L92) instead of here.

Reply:
Yeah I think so. Is there a reason that's a separate query? Shouldn't it be able to be dropped in right here?

Reply:
I added a separate [query](https://github.com/SeedCompany/cord-api-v3/blob/e6113e787bb30618ab4cc5ce30595db9f9cb045c/src/components/engagement/handlers/apply-finalized-changeset-to-engagement.handler.ts#L80~L96) for not looping nodes that are created in changeset but I think we can just do it here.

Reply:
I think so too